video/out/d3d11/context.c

```diff
@@ -203,6 +204,12 @@ static bool d3d11_start_frame(struct ra_swapchain *sw, struct ra_fbo *out_fbo)
     if (!out_fbo)
         return true;
 
+    if (p->backbuffer) {
+        MP_ERR(sw->ctx, "`start_frame` called while a backbuffer was hanging!\n");
```

Comment:
The word "hanging" doesn't make much sense to me here. Maybe this should just be something like "`start_frame` called while a frame was in progress" or "`start_frame` called without `submit_frame`", eg. about the API contract that was broken, rather than the backbuffer.

Reply:
I would just `assert`. It's an internal API.

Reply:
Made this an assert as yes, it's an internal API.